sql/deployments/deployment-2018-01-16-COLAB-2495.sql

```diff
@@ -0,0 +1,18 @@
+# Display all entries that have the issue
+SELECT * FROM xcolab.members_Member WHERE members_Member.status IS  NULL ;
+
+# Display entries without a createDate and the replacement createDate (DryRun)
+SELECT T1.id_, T1.screenName, T1.createDate, T2.createDate
+FROM xcolab.members_Member T1, xcolab.members_Member T2
+WHERE T1.id_ = T2.id_ + 1 AND T1.createDate IS NULL;
```

Comment:
It's good to have the SQL statements for reference, but I would comment them out so they're not run when we pipe this file to mysql.